packages/@uppy/svelte/package.json

```
@@ -10,7 +10,7 @@
   },
   "exports": {
     ".": {
-      "svelte": "./src/index.js",
+      "svelte": "./lib/index.js",
       "default": "./lib/index.js"
     },
```

No need to provide a map if it should always resolve to the same file
```suggestion
    ".": "./lib/index.js",
```
Alternatively, we can follow SvelteKit documentation recommendation:
```suggestion
    ".": {
      "types": "./lib/index.d.ts",
      "svelte": "./lib/index.js"
    },
```